ci/snap/after_success.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+
+RESULT_SNAP=$(find ./ -name "*.snap")
+
+sudo snap install "$RESULT_SNAP" --dangerous --classic
+
+/snap/bin/neovim --version
```

Comment:
why is the binary named `neovim`? that is confusing, it should be named `nvim`.

Reply:
@justinmk,

The snap package is called neovim, so the binary is called neovim. You can ask for an alias, but that is controlled within the snapstore. For now

```
snap alias neovim.neovim nvim
```

If you wish, I can change the package name to nvim.